src/Microsoft.DocAsCode.Common/Loggers/ReportLogListener.cs

```diff
@@ -23,7 +23,8 @@ public ReportLogListener(string reportPath, string repoRoot, string root)
             {
                 Directory.CreateDirectory(dir);
             }
-            _writer = new StreamWriter(reportPath, true);
+            var fs = new FileStream(reportPath, FileMode.OpenOrCreate, FileAccess.Write, FileShare.Read);
+            _writer = new StreamWriter(fs);
```

Comment:
>StreamWriter [](start = 26, length = 12)

do we really need to change this line? 

according to reference source, `new StreamWriter(reportPath, true)` == `new FileStream(reportPath, FileMode.Append, FileAccess.Write, FileShare.Read)`
[https://referencesource.microsoft.com/#mscorlib/system/io/streamwriter.cs,226](https://referencesource.microsoft.com/#mscorlib/system/io/streamwriter.cs,226)

Reply:
Reversed.